src/Reader/Extension/Podcast/Entry.php

```diff
@@ -129,6 +129,28 @@ public function getKeywords()
         return $this->data['keywords'];
     }
 
+    /**
+     * Get the entry title
+     *
+     * @return string
+     */
+    public function getTitle()
```

Comment:
Mmmmh, a test class for the entire `Zend\Feed\Reader\Extension\Podcast\Entry` is missing.